### ISSUE TITLE
Resource Management - UI

### ISSUE THREAD

**Issue by Ardianpalloshi** [amosproj/amos2022ws05-shared-desk-mgmt#19]
## User story
1. As a user (admin)
2. I want / need to be able to see all resources added by my company and enter the information for new ones
3. So that I can add new ones, edit or delete them

## Acceptance criteria
* The resource management page should only be visible for admins
* The resource management page should provide a list of all available resources of the admin's company with their respective properties
* The resource management page should offer a button for editing and one for deleting a resource as well as an option for adding a new resource
* The resource management page should offer a search field to quickly find certain resources
* The page should offer options to filter the displayed resources depending on certain properties, such as country, location, room and type and update the displayed list of resources accordingly
* After hitting the add resource button a page should be displayed where an admin can enter the necessary information for a new resource; the page should also contain a save/add button to offer access to the add ressource function
* After hitting the edit resource button a page should be displayed where an admin can update the information of a resource; this page should also contain a save button to offer access to the edit resource function

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Ardianpalloshi** [comment]
![Image](https://user-images.githubusercontent.com/115380013/203553610-33f0b6ba-bfac-49a6-a984-f2a18c2b5e9d.png)
